packages/sdk-codegen/src/python.gen.ts

```diff
@@ -263,9 +278,28 @@ ${this.hooks.join('\n')}
   }
 
   initArg(indent: string, property: IProperty) {
-    return `${indent}self.${property.name} = ${property.name}`
+    let propName = property.name
+    if (this.pythonKeywords.has(propName)) {
+      propName = propName + '_'
+    }
+    return `${indent}self.${propName} = ${propName}`
```

Comment:
discovered this bug when `LookmlModelExploreJoins` got a constructor (it was getting `self.from = from` which is not valid python syntax)

Reply:
Fwiw, I created a `reserve(name)` function in some of the other generators to abstract the "fix this reserved word" problem

Reply:
yeah, I was gonna dig more into how we're doing this in python when I look into implementing the kebab stuff. I'd like to try to make it conform more to what you have for the other language generators